desk/app/reel.hoon

```diff
@@ -238,6 +241,16 @@
     :~  [%pass +.pole %agent dock %poke cage]
         [%pass /expire/[ship.pole]/[token.pole] %arvo %b [%wait (add ~h1 now.bowl)]]
     ==
+  ::
+      [%v1 %id-link id=*]
+    =/  id  (crip +:(spud id.pole))
+    ?~  token=(~(get by stable-id) id)  `this
+    ?:  (~(has in open-describes) u.token)
+      ::  when the confirmation comes back we'll send the fact
+      `this
```

Comment:
And we don't care whether we already have a relevant entry in `our-metadata` because that'll get overwritten once the open describe comes back?

Reply:
right